### PR TITLE
[REF] CONTRIBUTING: Consider post_load for monkey patches

### PR DESCRIPTION
**Pull request by moylop260**
See example https://github.com/OCA/server-tools/pull/1340
Continuation from https://github.com/OCA/maintainer-tools/pull/196

Reduce the risks about issues related like: https://github.com/OCA/server-tools/issues/1335

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -183,12 +185,14 @@ Remember to add into the **`__init__.py`** the following imports as
...
 ```python
 ...
-from .hooks import pre_init_hook
-from .hooks import post_init_hook
-from .hooks import uninstall_hook
+from .hooks import pre_init_hook, post_init_hook, uninstall_hook, post_load
 ...
 ```
 
+For apply monkey patches use post_load hook.
```
Comment (pedrobaeza) [inline CONTRIBUTING.md]: s/apply/applying

**Comment by moylop260** [comment]
Thanks for review @pedrobaeza 
Typo fixed

About [models._patch_method](https://github.com/odoo/odoo/blob/75630e3d69fe6466f32edf9b8fa2abdfc20abdb5/odoo/models.py#L4260) and [models._revert_method](https://github.com/odoo/odoo/blob/75630e3d69fe6466f32edf9b8fa2abdfc20abdb5/odoo/models.py#L4291) are patches for methods of [models.model](https://github.com/odoo/odoo/blob/75630e3d69fe6466f32edf9b8fa2abdfc20abdb5/odoo/models.py#L5195) class and these ones don't require a post_load patch. In fact, you can use these patch methods from the definition of your custom model class.

**Comment by pedrobaeza** [comment]
Do you know why Travis is red?

**Comment by moylop260** [comment]
I just rebuilt the last build of the stable branch and It is red too:

The message error is `setuptools 40.0.0 Requirement.parse('setuptools<31')`:
 - https://travis-ci.org/OCA/maintainer-tools/jobs/412134067#L518

This is an unrelated issue for this PR.

Any clues @sbidoul?

**Comment by sbidoul** [comment]
I fixed the travis issue in #368

**Comment by moylop260** [comment]
Now, It is green.

**Pull request merged**